lib/repack.c

```diff
@@ -112,8 +122,18 @@ must_be_superuser(const char *func)
 #define RENAME_REL(relid, newrelname) RenameRelationInternal(relid, newrelname, PG_TOAST_NAMESPACE);
 #elif PG_VERSION_NUM < 90300
 #define RENAME_REL(relid, newrelname) RenameRelationInternal(relid, newrelname);
-#else
+#elif PG_VERSION_NUM < 120000
 #define RENAME_REL(relid, newrelname) RenameRelationInternal(relid, newrelname, true);
+#else
+#define RENAME_REL(relid, newrelname) RenameRelationInternal(relid, newrelname, true, false);
+#endif
+/*
+ * is_index flag was added in 12.0, prefer separate macro for relation and index
+ */
+#if PG_VERSION_NUM < 120000
+#define RENAME_INDEX(relid, newrelname) RENAME_REL(relid, newrelname);
+#else
+#define RENAME_INDEX(relid, newrelname) RenameRelationInternal(relid, newrelname, true, false);
```

Comment:
Maybe this is just nitpicking, but why the fourth argument is false if we use the `RenameRelationInternal()` to rename index?

Reply:
Ouch! Thank you, this is really my error.